packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -66,7 +66,9 @@ const booksDefaultIntegration = new LambdaProxyIntegration({
   handler: booksDefaultFn,
 });
 
-const httpApi = new HttpApi(stack, 'HttpApi');
+const httpApi = new HttpApi(stack, 'HttpApi', {
+    decscription: 'My books API',
+});
```

Comment:
Revert this change. I've added a label to inform the linter to not complain.

Reply:
Ah great :D i was wondering